edge-agent/docker/linux/arm64v8/base/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get update && \
     ln -s /lib/aarch64-linux-gnu/libdl.so.2 /usr/lib/aarch64-linux-gnu/libdl.so && \
     ln -s /lib/aarch64-linux-gnu/libc.so.6 /usr/lib/aarch64-linux-gnu/libc.so.6
 
-# create a user to allow agent to optionally run as non-root
+# Create a user to allow installing the trustbundle under a user context
```

Comment:
This was needed for K8s. I thought we planned to clarify it.

Reply:
Given we'll be using the non-root user for single device, this comment without k8s specified is ok now.